CRM/Event/Form/ManageEvent/Registration.php

```diff
@@ -161,10 +160,10 @@ public function setDefaultValues() {
     }
 
     // provide defaults for required fields if empty (and as a 'hint' for approval message field)
-    $defaults['registration_link_text'] = CRM_Utils_Array::value('registration_link_text', $defaults, ts('Register Now'));
-    $defaults['confirm_title'] = CRM_Utils_Array::value('confirm_title', $defaults, ts('Confirm Your Registration Information'));
-    $defaults['thankyou_title'] = CRM_Utils_Array::value('thankyou_title', $defaults, ts('Thank You for Registering'));
-    $defaults['approval_req_text'] = CRM_Utils_Array::value('approval_req_text', $defaults, ts('Participation in this event requires approval. Submit your registration request here. Once approved, you will receive an email with a link to a web page where you can complete the registration process.'));
+    $defaults['registration_link_text'] ??= ts('Register Now');
```

Comment:
We can't use this operator yet since it was introduced in 7.4 and we're still supporting 7.3.

Reply:
Fixed, I thought I checked and we required 7.4, but I checked again and see we only recommend 7.4.